src/test/java/com/hocs/test/glue/foi/CaseCreationStageStepDefs.java

```diff
@@ -24,4 +26,16 @@ public void iEditTheCaseDetailsValue(String value) {
     public void theDetailsEnteredAreDisplayed() {
         caseCreationStage.assertCaseDetailsAreCorrect();
     }
+
+    @And("I fill the acknowledgement response date")
+    public void iFillTheAcknowledgementResponseDate() {
+        String responseDate = getTodaysDate();
```

Comment:
I dont think we need this line. We can just instantiate the string on the next line and not overwrite the value ascribed to it.